### DISCUSSION TITLE
Cron seems to not work : set to test every 120 minutes but runs every 60.

### DISCUSSION THREAD

**Discussion by stephdeville** [alexjustesen/speedtest-tracker#923] [Q&A] [answered]
Hi All, 

Everything is running smoothly ... and I'm really happy with  speedtest-tracker ... so first, thank you to all the contributors. 

Now, I just want to raise a topic here : 
I did have my test running every hour ... and wanted to change it to 2 hours but then noticed that even if that setting was saved in to the GUI, the tests continue to run every hour. 

Does anyone have a similar issue?

What i did so far ... updated to the very last release (0.13.3) : no luck. 
Disabled the schedule, saved ... and set the cron to */120 * * * * and saved again : no luck. 

Any suggestion? 

Thank you 

Stéphane

<img width="610" alt="2023-11-23_17-49-35" src="https://github.com/alexjustesen/speedtest-tracker/assets/6465135/70f71780-4d56-4e87-b68e-f096b8744b50">

**Comment by alexjustesen** [comment, marked as answer]
Thanks Stephane!

Can you give the expression `0 */2 * * *` a try? Both are technically correct so I'd be interested if the cron validator is having issues with minutes larger than 60.

**Reply by stephdeville** [reply]
Hi Alex, 

Your suggestion did solve my issue. 

Thank you 

Stéphane
